Feature/package setup

Adentas opened the pull request:
Доробив, протестував, зверни увагу що в main, commands, та інших змінився лише імпорт там додалась крапка перед назвою.

Adentas committed:
Не завершена версія

зберігаю щоб не зникли оновлення, не потребує ревью, та по'єднання

Adentas committed:
Package setup

Дороблений

UkrainianEagleOwl committed:
Merge branch 'main' into feature/package_setup

UkrainianEagleOwl reviewed (changes requested):
Треба передивитись нові файли notes_core і якщо ще є щоб там теж додати точки якщо потрібно і додати їх у маніфест, після цього вже буде ок зливати

Adentas committed:
Оновлений setup

Оновив сетап тепер він працює на новій версії бота, додав requires, а також виправив декілька помилок пов'язаних з бібліотеками

Adentas reviewed (commented):
Це остання версія
